### PR TITLE
import-indicator-commands

### PR DESCRIPTION
**Pull request by sapirshuker**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-6437

## Description
- Added the *tags_tlp*, *default_state*, *expiration_ts* arguments to the ***threatstream-import-indicator-with-approval*** command.
- Added the *tags_tlp* argument to the ***threatstream-import-indicator-without-approval*** command.
- Updated the docker image to: *demisto/py3-tools:1.0.0.63856*.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [X] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [X] Tests
- [X] Documentation

### CODE REVIEW

File: Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.yml [hunk 1]
```diff
@@ -2563,13 +2562,42 @@ script:
...
+      name: tags_tlp
+      predefined:
+      - Red
+      - Amber
+      - Green
+      - White
+      required: false
+      secret: false
+    - default: false
+      description: The time stamp when intelligence will expire on ThreatStream, in ISO format. For example, 2020-12-24T00:00:00. By default, the expiration_ts is set to 90 days from the current date.
```
Comment (ShirleyDenkberg) [inline Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.yml:2583]:
```suggestion
      description: The timestamp when intelligence will expire on ThreatStream, in ISO format. For example, 2020-12-24T00:00:00. By default, the expiration_ts is set to 90 days from the current date.
```

File: Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/README.md [hunk 2]
```diff
@@ -1478,7 +1479,10 @@ Imports indicators (observables) into ThreatStream. The imported data must be ap
...
 | email_mapping | Indicator type to assign if a specific type is not associated with an observable. This is a global setting that applies to any imported email-type observable when an explicit itype is not specified for it. | Optional | 
 | md5_mapping | Indicator type to assign if a specific type is not associated with an observable. This is a global setting that applies to any imported MD5-type observable when an explicit itype is not specified for it. | Optional | 
-| tags | A comma-separated list of tags. For example, tag1,tag2. | Optional | 
+| tags | A comma-separated list of tags applied to the imported observables. For example, tag1,tag2. | Optional | 
+| tags_tlp | You can add tags that are private to your organization by setting the tlp attribute for the tag to red. If you do not specify a tlp setting, the tag is visible to any ThreatStream user with access to the observable. Possible values are: Red, Amber, Green, White. | Optional | 
+| expiration_ts | The time stamp when intelligence will expire on ThreatStream, in ISO format. For example, 2020-12-24T00:00:00. By default, the expiration_ts is set to 90 days from the current date. | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/README.md:1484]:
```suggestion
| expiration_ts | The timestamp when intelligence will expire on ThreatStream, in ISO format. For example, 2020-12-24T00:00:00. By default, the expiration_ts is set to 90 days from the current date. | Optional | 
```

File: Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/README.md [hunk 3]
```diff
@@ -1524,14 +1528,15 @@ Imports indicators (observables) into ThreatStream. Approval is not required for
 | --- | --- | --- |
 | confidence | The observable certainty level of a reported indicator type. Default is 50. | Optional | 
 | source_confidence_weight | Ratio (0-100) between the source confidence and the ThreatStream confidence. To use your specified confidence entirely and not re-assess the value using machine learning algorithms, set this argument to 100. | Optional | 
-| expiration_ts | The time stamp when intelligence will expire on ThreatStream, in ISO format. For example, 2020-12-24T00:00:00. | Optional | 
-| severity | The severity to assign to the observable when it is imported. Can be "low", "medium", "high" , or "very-high". Possible values are: low, medium, high, very-high. | Optional | 
-| tags | A comma-separated list of tags. For example, tag1,tag2. | Optional | 
+| expiration_ts | The time stamp when intelligence will expire on ThreatStream, in ISO format. For example, 2020-12-24T00:00:00. By default, the expiration_ts is set to 90 days from the current date. | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/README.md:1531]:
```suggestion
| expiration_ts | The timestamp when intelligence will expire on ThreatStream, in ISO format. For example, 2020-12-24T00:00:00. By default, the expiration_ts is set to 90 days from the current date. | Optional | 
```

File: Packs/Anomali_ThreatStream/ReleaseNotes/2_1_6.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+
+##### Anomali ThreatStream v3
+
+- Added the *tags_tlp*, *default_state*, *expiration_ts* arguments to the ***threatstream-import-indicator-with-approval*** command.
```
Comment (ShirleyDenkberg) [inline Packs/Anomali_ThreatStream/ReleaseNotes/2_1_6.md:6]:
```suggestion
- Added the following arguments to the ***threatstream-import-indicator-with-approval*** command:
    - *tags_tlp*
    - *default_state*
    - *expiration_ts*
```

File: Packs/Anomali_ThreatStream/ReleaseNotes/2_1_6.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+- Added the *tags_tlp* argument to the ***threatstream-import-indicator-without-approval*** command.
+- Updated the docker image to: *demisto/py3-tools:1.0.0.63856*.
```
Comment (ShirleyDenkberg) [inline Packs/Anomali_ThreatStream/ReleaseNotes/2_1_6.md:8]:
```suggestion
- Updated the Docker image to: *demisto/py3-tools:1.0.0.63856*.
```

**Comment by ShirleyDenkberg** [comment]
@Shellyber Doc review completed.

**Review by Shellyber — APPROVED** [review]
Nice

**Pull request merged**